docker/px4-dev/Dockerfile_ros-kinetic

```diff
@@ -39,6 +40,8 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9F
 	&& apt-get clean autoclean \
 	# pip
 	&& pip install --upgrade numpy px4tools pymavlink \
+	&& pip3 install --upgrade setuptools wheel \
```

Comment:
I'm still not sure why this line is necessary

Reply:
@lamping7  I'm also not sure, but if you remove that line it will complain it can't find setuptools and wheel

Reply:
Makes sense.  In the base container we install the apt package for python2 to resolve these.

Reply:
ok, I see. Generally, it seems to be recommended to always take the packages from pypi instead of the packages supplied by apt for python...

Reply:
Generally yes. The reason is to better handle dependencies and updating. Depending on how the containers are used this isn't really an issue, like in the case of CI where they're created and thrown away after a test or build.